packages/expo-in-app-purchases/src/InAppPurchases.types.ts

```diff
@@ -0,0 +1,63 @@
+export type ValidItemType = 'inapp' | 'subs';
+
+export interface QueryResponse {
```

Comment:
Choose names that make sense in a global context so that the application code is easier to read with less context:

```suggestion
export interface IAPQueryResponse {
```

Reply:
I agree that's a good idea for most of these but I think for the constants (ResponseCodes and ErrorCodes) it might get repetitive and seem redundant if a user does a * import. For example:
```javascript
import * as InAppPurchases from 'expo-in-app-purchases';

// Later...
if (responseCode === InAppPurchases.IAPResponseCode.OK) {}
```
so the IAP prefix might not be ideal for `ResponseCode` and `ErrorCode` . Maybe I can borrow Android's name and call them "BillingResponseCodes" or something similar. I also want to avoid it being too verbose. Thoughts?